src/test/java/uk/gov/ons/census/casesvc/service/BlankQuestionnaireServiceTest.java

```diff
@@ -384,7 +383,6 @@ private String getQid(String qidType) {
   }
 
   @AllArgsConstructor
-  @EqualsAndHashCode
```

Comment:
How is your map going to work without equals and hashcode on the key?

Reply:
The fact that you can remove this indicates that the unit tests are broken/flawed

Reply:
There's no Map in the unit tests.
The Key class is a data class to set up the parameters for the tests.